### PR TITLE
Created Review entity

### PR DESCRIPTION
**Pull request by md1guy**

### CODE REVIEW

**Review by Senchakmen — DISMISSED** [review]

**Review by onlyVone-zz — DISMISSED** [review]

File: src/main/java/com/softserve/actent/model/entity/Review.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NonNull
+    @NotBlank(message = "Review text can`t be empty.")
+    private String text;
+
+    @NonNull
+    @NotNull(message = "Review can`t be null.")
+    private Integer score;
```
Comment (RBarnabass) [inline src/main/java/com/softserve/actent/model/entity/Review.java:33]: nullable false

File: src/main/java/com/softserve/actent/model/entity/Review.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+@Entity
+@Table(name = "reviews")
+public class Review {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NonNull
+    @NotBlank(message = "Review text can`t be empty.")
+    private String text;
```
Comment (RBarnabass) [inline src/main/java/com/softserve/actent/model/entity/Review.java:28]: nullable false

File: src/main/java/com/softserve/actent/model/entity/Review.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "reviews")
+public class Review {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NonNull
```
Comment (RBarnabass) [inline src/main/java/com/softserve/actent/model/entity/Review.java:25]: Not null

File: src/main/java/com/softserve/actent/model/entity/Review.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    @NotBlank(message = "Review text can`t be empty.")
+    @Column(nullable = false)
+    private String text;
```
Comment (RBarnabass) [inline src/main/java/com/softserve/actent/model/entity/Review.java:28]: Mast be max lenght!

**Review by Senchakmen — APPROVED** [review]

**Review by RBarnabass — APPROVED** [review]

**Review by thebadpanda — APPROVED** [review]

**Pull request merged**